src/amo/components/AddonReviewList.js

```diff
@@ -107,6 +107,15 @@ export class AddonReviewListBase extends React.Component {
     return `${this.addonURL()}reviews/`;
   }
 
+  setReviewBody(review) {
```

Comment:
`reviewBody` or `reviewBodyHTML` would be better method names; this doesn't actually set anything.